cmd/serviceaccnt.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	serivceAccount "github.com/method-security/methodk8s/internal/serviceaccount"
+	"github.com/spf13/cobra"
+)
+
+func (a *MethodK8s) InitServiceAccountCommand() {
+	serviceAccountCmd := &cobra.Command{
+
+		Use:   "service-account",
```

Comment:
can we call it `serviceaccount` to make it match k8s?

Reply:
Only other flag is whether there's something else we should call it because down the road we very likely will have a `methodk8s serviceaccount enumerate` where we enumerate all service accounts in the cluster.